test/integration_test/lib/util/test_author.py

```diff
@@ -0,0 +1,12 @@
+import json
+from lib.util.author import *
+
+def test_get_uid_map():
```

Comment:
The test needs to start with a valid `clean_data.json` and match the generated `author_uid_map_test.json` against an expected `author_uid_map.json`.

The test module should cover 100% of the module it is trying to test. You can see the code coverage of this PR [here](https://codecov.io/gh/DeveloperCAP/MLCAT/pull/151/changes). The red lines are the ones that are not covered by the test(s).

Reply:
@tejalwakchoure You need to specify the complete path of `clean_data.json`. The test you wrote is not passing. See [this](https://travis-ci.org/DeveloperCAP/MLCAT/jobs/344345542)

You can refer to the tests written, how the import the files from `test/integration_test/data/`

Reply:
@prasadtalasila @hrishikesh-dahiya I have made some changes in this direction.